lib/common.js

```diff
@@ -7,6 +7,7 @@ var Table = require('cli-table');
 var ini = require('./utils/ini');
 var moment = require("moment");
 var _ = require("underscore");
+const DT_FORMAT = 'YYYY-MM-DD h:mm:ss:SSS';
```

Comment:
Hi @camilamacedo86 are all of the changes in this file to remove the deprecation notice?

Reply:
@davidffrench, in this case, is because of the type of value returned by the backend service, it is coming as timestamp and not date.  You can check it here; https://github.com/feedhenry/fh-fhc/blob/master/lib/cmd/fh3/appforms/environments/data-sources/audit-logs.js#L29

Reply:
Yes.